src/components/MarkdownPage/MarkdownPage.js

```diff
@@ -29,6 +29,16 @@ const MarkdownPage = ({
   markdownRemark,
   sectionList,
   titlePostfix = '',
+}: {
+  authors: Array<string>,
+  createLink: Function,
+  date: string,
+  enableScrollSync: boolean,
```

Comment:
`date` and `enableScrollSync` weren't marked as required before, and they are ommitted in some cases, so we should make them optional here..

Reply:
👍